app/controllers/application_controller.rb

```diff
@@ -388,4 +391,48 @@ def force_ssl
       redirect_to("https://#{request.host_with_port}#{request.fullpath}") unless request.ssl? || ( request.headers["HTTP_VIA"] && request.headers["HTTP_VIA"].include?("sharetribe_proxy")) || request.fullpath == "/robots.txt"
     end
   end
+
+  def feature_flags
```

Comment:
I am confused why we have the same method in here and FeatureFlagHelper?
We already included FeatureFlagHelper.